implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -204,14 +199,68 @@ The supported sizes are
   </tbody>
 </table>
 
+The same mapping will also be attempted for SKU level data
 
-The `priceDecorator` uses the currency formatting definitions of the Nosto account to format prices into `priceText` and `listPriceText` fields. The fields required for this mapping are
+### Price decorators
 
-* `price` will be formatted to `priceText`
-* `listPrice` will be formatted to `listPriceText`
-* `priceCurrencyCode` will be used as the currency code
+* **Include Required Fields**
+  The `priceDecorator` uses the currency formatting definitions of the Nosto account to format prices into `priceText` and `listPriceText` fields. The fields required for this mapping are:
```

Comment:
Move the first sentence above this section as an intro paragraph to this section?